packages/wonder-blocks-dropdown/src/components/__tests__/combobox.test.tsx

```diff
@@ -153,6 +153,29 @@ describe("Combobox", () => {
         expect(screen.getByRole("combobox")).toHaveValue("");
     });
 
+    it("should reset the input to its previous selected value when the user presses Escape", async () => {
```

Comment:
**note:** This test verifies one of the cases that you found in the previous PR, where the combobox wasn't being changed back to its previous selected label.